src/Activation/AcceptEula.tsx

```diff
@@ -55,43 +53,46 @@ const AcceptEula: FunctionComponent = () => {
     : t("label.unchecked_checkbox")
 
   return (
-    <GradientBackground gradient={Colors.gradientPrimary10}>
-      <SafeAreaView style={style.container}>
-        <GlobalText style={style.headerText}>
-          {t("onboarding.terms_header_title")}
-        </GlobalText>
-        <EulaLink
-          docName={"onboarding.privacy_policy"}
-          onPress={linkToPrivacyPolicy}
-        />
-        <EulaLink docName={"onboarding.eula"} onPress={linkToEula} />
-        <View style={style.footerContainer}>
-          <TouchableOpacity
-            style={style.checkboxContainer}
-            onPress={() => toggleCheckbox(!boxChecked)}
-            accessible
-            accessibilityRole="checkbox"
-            accessibilityLabel={checkboxLabel}
-            testID="accept-terms-of-use-checkbox"
-          >
-            <SvgXml
-              xml={checkboxIcon}
-              fill={Colors.primary100}
-              width={Iconography.small}
-              height={Iconography.small}
-            />
-            <GlobalText style={style.checkboxText}>
-              {t("onboarding.eula_agree_terms_of_use")}
-            </GlobalText>
-          </TouchableOpacity>
-          <Button
-            onPress={handleOnPressNext}
-            disabled={!boxChecked}
-            label={t("common.continue")}
+    <>
+      <StatusBar backgroundColor={Colors.primaryLightBackground} />
```

Comment:
Just asking, does this mean that we will want to manually render a StatusBar on every screen where we want one, rather than having it be the default and then trying to hide it via react-native?

Reply:
I believe the status bar will show up anyways, but on most screens we will still want to to use `useStatusBarEffect` and on many we will want to use `StatusBar` to achieve the correct styling.

Reply:
If that doesn't answer your question lmk and I can explain more clearly!